web/web.go

```diff
@@ -342,17 +344,32 @@ func New(logger log.Logger, o *Options) *Handler {
 	return h
 }
 
-var corsHeaders = map[string]string{
-	"Access-Control-Allow-Headers":  "Accept, Authorization, Content-Type, Origin",
-	"Access-Control-Allow-Methods":  "GET, OPTIONS",
-	"Access-Control-Allow-Origin":   "*",
-	"Access-Control-Expose-Headers": "Date",
-}
-
 // Enables cross-site script calls.
-func setCORS(w http.ResponseWriter) {
-	for h, v := range corsHeaders {
-		w.Header().Set(h, v)
+func setCORS(w http.ResponseWriter, o []string, r *http.Request) {
```

Comment:
I don't like that we're duplicating security related code. Could you move this under pkg somewhere?

Reply:
will `/util/httputil` package be a better place to put this? I am not sure which will be the best place under `pkg`

Reply:
That sounds good.